### ISSUE TITLE
Good Package but not enough documentation

### ISSUE THREAD

**Issue by therimalaya** [zachcp/msaR#3]
I am building a shiny app where I am using this msaR. The documentation referes directly to msa library of javascript where I can see what are the possible options but the documentation in this side is not enough. For example, I am trying to add a selection on the output but don't know how to pass the option to javascript from given R interface. Although there are few indication of passing options like config and zoomer but all the example are a key-value based which are easy to input. It is rather difficult to input selection with some search criteria or start or end of column.

An updated document will certainly help a lot.

**Comment by zachcp** [comment]
I agree a bit of documentation could help but there is not much more to document that is not on the website. This is a one-way app right now - you can display MSAs in the browser and you have some display options. You can access all of the features of the underlying JS library or you can limit/change them.

To work with Shiny interactively you would need more JS interop than there currently is. You would need to specify some JS event and send it back to R. Right now this does not happen, althouhg it is certainly possible. Do you have anything particular in mind?   I think mouse-selection of sequences could be nice.  

What does this sentence mean: "input selection with some search criteria or start or end of column"? What behavior do you want to achieve? Select input before it is displayed? Afterwards?

There is lots of room for improvement but I need a bit more info to be helpful.

zach cp

**Comment by therimalaya** [comment]
Thank you zachcp for the reply. I am thinking If I have a start position and end position and I want to highlight the part of MSA sequence using these position. All these things I should be able to do programatically. There is already some option using the toolbar in plot and option in JavaScript but I am not sure how a R user can send such option and values in option from R.
